cmd/create/oidcconfig/cmd.go

```diff
@@ -150,30 +169,58 @@ func run(cmd *cobra.Command, argv []string) {
 		}
 	}
 
-	if !args.rawFiles {
-		r.Reporter.Infof("This command will create a S3 bucket populating it with documents " +
-			"to be compliant with OIDC protocol. It will also create a Secret in Secrets Manager containing the private key.")
+	if args.managed && mode != aws.ModeAuto {
```

Comment:
![Screenshot from 2023-03-24 14-27-22](https://user-images.githubusercontent.com/118839428/227609909-38188ed0-9c14-48cf-8441-6298a56d831d.png)
Tried running the condition via CLI but not getting the warning. Am I approaching this incorrectly? Please kindly advise. @gdbranco